sdk/core/core-amqp/package.json

```diff
@@ -44,6 +44,7 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-amqp",
+  "sideEffects": false,
```

Comment:
This seems to be a setting mostly used by WebPack. Since rollup doesn't use it and has it's own mechanism, doesn't seem like we should be adding this. More context -https://github.com/rollup/rollup/issues/2593
Deferring to @bterlson for any other insights

Reply:
Our customers may be using the package with webpack, which is why it can be useful to have this field.

Reply:
Yes this is for customer bundling more than our own. Webpack is far and away the most popular tool for this.